docs/src/tutorials/supported_operations.md

```diff
@@ -16,10 +16,11 @@ On `thin-edge.io` the support for one such operation can be added using the `thi
 ### `thin-edge.io` Supported Operations API
 
 The Supported Operations utilises the file system to add and remove operations. A special file placed in `/etc/tedge/operations` directory will indicate that an operation is supported.
-The specification for the operation files is described in `thin-edge.io` specifications repository[src/supported-operations/README.md](https://github.com/thin-edge/thin-edge.io-specs/blob/main/src/supported-operations/README.md)
+The specification for the operation files is described in `thin-edge.io` specifications repository [src/supported-operations/README.md](https://github.com/thin-edge/thin-edge.io-specs/blob/main/src/supported-operations/README.md)
```

Comment:
Unrelated to this PR, but surprised to see a link to the specs repo, for a feature that we've exposed to the end-user. This could be replaced with a local reference to the `Adding new custom operation` section in this document itself.

Reply:
I was surprised too and created that issue https://github.com/thin-edge/thin-edge.io/issues/1547